admin/templates/Includes/LeftAndMain_Menu.ss

```diff
@@ -1,36 +1,10 @@
 <div class="cms-menu cms-panel cms-panel-layout west" id="cms-menu" data-layout-type="border">
 	<div class="cms-logo-header north">
-		<div class="cms-logo">
-			<a href="$ApplicationLink" target="_blank" title="$ApplicationName (Version - $CMSVersion)">
-				$ApplicationName <% if $CMSVersion %><abbr class="version">$CMSVersion</abbr><% end_if %>
-			</a>
-			<span><% if $SiteConfig %>$SiteConfig.Title<% else %>$ApplicationName<% end_if %></span>
-		</div>
-	
-		<div class="cms-login-status">
-			<a href="$LogoutURL" class="logout-link font-icon-logout" title="<%t LeftAndMain_Menu_ss.LOGOUT 'Log out' %>"></a>
-			<% with $CurrentMember %>
-				<span>
-					<%t LeftAndMain_Menu_ss.Hello 'Hi' %>
-					<a href="{$AbsoluteBaseURL}admin/myprofile" class="profile-link">
-						<% if $FirstName && $Surname %>$FirstName $Surname<% else_if $FirstName %>$FirstName<% else %>$Email<% end_if %>
-					</a>
-				</span>
-			<% end_with %>
-		</div>
+		<% include LeftAndMain_MenuStatus %>
 	</div>
-		
-	<div class="cms-panel-content center">
-		<ul class="cms-menu-list">
-		<% loop $MainMenu %>
-			<li class="$LinkingMode $FirstLast <% if $LinkingMode == 'link' %><% else %>opened<% end_if %>" id="Menu-$Code" title="$Title.ATT">
-				<a href="$Link" $AttributesHTML>
-					<span class="icon icon-16 icon-{$Code.LowerCase}">&nbsp;</span>
-					<span class="text">$Title</span>
-				</a>
-			</li>
-		<% end_loop %>
-		</ul>
+
+    <div class="cms-panel-content center">
+		<% include LeftAndMain_MenuList %>
 	</div>
 		
 	<div class="cms-panel-toggle south">
```

Comment:
In the case where you wanted to remove/customise it then you would have to customise this file rather than just the include - I think it makes sense as it is a "component" not a required piece of code?